### DISCUSSION TITLE
nzyme-tap_2.0.0-alpha.6_rpios11_arm64 (hangs after 2 days of operation)

### DISCUSSION THREAD

**Discussion by a020088** [nzymedefense/nzyme#994] [Questions]
Hi,
I am following nzyme now since years I believe starting from 1.0.0 onwards. I am very interested in this app.

However since I am using nzyme 2.0.0 I have problems with stability.

I am able to install it completely (now alpha 6) and let it run.
I installed node and taps on a single raspi 4 with SSD connected.
I have two antennas installed.

Unfortunately I have to say the the raspi stops working after a while (I guess after 2 days).
Can you please help me,

Is this a performance issue or what can I do to make it run longer.

Thanks for your support

**Comment by lennartkoopmann** [comment]
What do you mean with "stops working"?

**Reply by a020088** [reply]
Have I said : Stops working.
To be honest this I cannot confirm. 
The problem is that I cannot login anymore: neither via ssh on raspi nor on Nzyme?
Therefore I have thought maybe overloaded by taps or nzyme engine?
Is there something I can do?
When using “top” I think I have seen cpu-load > 100%.
Thanks for support.

**Reply by lennartkoopmann** [reply]
The first step would be to find out what exactly is happening. Try to look at the system logs and dmesg output to find out what the system did before it crashed.

If the system appears to be up, but unresponsive, you should check for CPU and memory usage. Nzyme should run on a RPI 4 without stability issues.
